tests/test_cli/dash_test_script.txt

```diff
@@ -0,0 +1,5 @@
+# from manim import *
+class Dash_Test(Scene):
+        def construct(self):
```

Comment:
nitpick: this file uses indentation that is 8 spaces wide, please change to 4, if possible. (I know this is not actual code, but just for the sake of consistency...)